test/e2e/run_networking_test.go

```diff
@@ -396,12 +396,20 @@ var _ = Describe("Podman run networking", func() {
 		name := "testctr"
 		session := podmanTest.Podman([]string{"create", "-t", "--expose", "80", "-p", "80", "--name", name, ALPINE, "/bin/sh"})
 		session.WaitWithDefaultTimeout()
+		Expect(session).Should(ExitCleanly())
 		inspectOut := podmanTest.InspectContainer(name)
 		Expect(inspectOut).To(HaveLen(1))
 		Expect(inspectOut[0].NetworkSettings.Ports).To(HaveLen(1))
 		Expect(inspectOut[0].NetworkSettings.Ports["80/tcp"]).To(HaveLen(1))
-		Expect(inspectOut[0].NetworkSettings.Ports["80/tcp"][0].HostPort).To(Not(Equal("80")))
+		hostPort := inspectOut[0].NetworkSettings.Ports["80/tcp"][0].HostPort
+		Expect(hostPort).To(Not(Equal("80")))
 		Expect(inspectOut[0].NetworkSettings.Ports["80/tcp"][0]).To(HaveField("HostIP", "0.0.0.0"))
+
+		session = podmanTest.Podman([]string{"ps", "-a", "--format", "{{.Ports}}"})
+		session.WaitWithDefaultTimeout()
+		Expect(session).Should(ExitCleanly())
+		// This must use Equal() to ensure we do not see the extra ", 80/tcp" from the exposed port
+		Expect(session.OutputToString()).To(Equal("0.0.0.0:" + hostPort + "->80/tcp"))
```

Comment:
Should we have a test with a port range?

Reply:
I added sufficient unit tests IMO, but I can add more int test if wanted. I do agree that expose with ranges seems to have very little e2e tests (one, two if we could the ones form the image)

Reply:
added more tests